atd-vzd/migrations/default/1714576842105_delete_crufty_locations_columns/up.sql

```diff
@@ -0,0 +1,51 @@
+-- This is an old crufty view that needs to be dropped bc it has dependencies with some columns we are dropping
+DROP VIEW atd_txdot_locations_with_centroids;
```

Comment:
this view can be dropped it is cruft and not used anywhere! i inventoried it [here](https://docs.google.com/spreadsheets/d/1oHzKt8Cvd1BGpZ7KJl4dfBDHGuSl7gs4yN0OM7AJMBA/edit#gid=1171053547)